src/api/src/main/java/engineering/everest/lhotse/api/rest/controllers/AdminOrganizationsController.java

```diff
@@ -51,34 +53,33 @@ public AdminOrganizationsController(DtoConverter dtoConverter,
     @ApiOperation("Retrieves details of all organizations")
     @AdminOnly
     public List<OrganizationResponse> getAllOrganizations() {
-        return organizationsReadService.getOrganizations().stream()
-                .map(dtoConverter::convert)
-                .collect(toList());
+        return organizationsReadService.getOrganizations().stream().map(dtoConverter::convert).collect(toList());
     }
 
     @PostMapping
     @ResponseStatus(CREATED)
     @ApiOperation("Register a new organization")
     @AdminOnly
-    public UUID newOrganization(User requestingUser, @RequestBody @Valid NewOrganizationRequest request) {
-        return organizationsService.createRegisteredOrganization(requestingUser.getId(), request.getOrganizationName(),
-                request.getStreet(), request.getCity(), request.getState(), request.getCountry(), request.getPostalCode(),
-                request.getWebsiteUrl(), request.getContactName(), request.getContactPhoneNumber(), request.getContactEmail());
+    public UUID newOrganization(@ApiIgnore Principal principal, @RequestBody @Valid NewOrganizationRequest request) {
+        return organizationsService.createOrganization(UUID.fromString(principal.getName()),
```

Comment:
The flow for an application admin to create an organisation, a user and then promoting them to an org admin has been left out.

(Its only supported for the self-registration case)

Reply:
We are replacing the Jupyter notebook as a way of documenting the flow.

An issue on the backlog to add some detailed text explaining the contrived example would be worth doing. I've created an issue for this: https://github.com/everest-engineering/lhotse/issues/52

You'll be happy to hear it will be in Python this time ;-)